python/inline/conftest.py

```diff
@@ -1034,6 +1042,25 @@ def reportinfo(self) -> Tuple[Union["os.PathLike[str]", str], Optional[int], str
 
 
 class InlinetestModule(pytest.Module):
+    def order_tests(test_list, tags):
+        prio_unsorted = []
+        unordered = []
+
+        # sorting the tests based if they are ordered or not
+        for test in test_list:
+            if test.tag in tags:
+                prio_unsorted.insert(tags.index(test.tag), test)
```

Comment:
I'm not sure if this line assumes that there is only at most one test with the same tag? Maybe you can add a test where two inline tests have the same tag to check if that could be a problem. Example:
```
Here(tag=["a"])... # t1
Here(tag=["b"])... # t2
Here(tag=["a"])... # t3

--inlinetest-order=b --inlinetest-order=a -> correct order should be t2, t1, t3 or t2, t3, t1
```